.travis.yml

```diff
@@ -11,34 +11,22 @@ cache:
     - rust/target  # cached objects for Rust library code
     - examples/target  # cached objects for Rust example code
 
-env:
-  - FORMAT_CHECKS=true
-  - BUILD_SERVER=true
-  - BUILD_EXAMPLES=true
-  - UNIT_TEST=true
-  - EXAMPLES_TEST=true
-
-script:
-  - echo "build --show_progress_rate_limit=5.0" >> .bazelrc
-  - set -o errexit
```

Comment:
Is this not needed anymore?

Reply:
I am not entirely sure, so I added it back for now. @daviddrysdale can confirm whether it is actually needed.

Reply:
Not sure &ndash; you could check by taking these 2 lines out and:
 - seeing if the Travis log gets truncated (due to excessive Bazel output)
 - put in a deliberate failure and check the Travis job fails.

Reply:
I will submit as it is, and then try and remove it in a future PR.